### PR TITLE
Multiplestep

### PR DESCRIPTION
**Pull request by cornflake15**

### CODE REVIEW

**Comment by cornflake15** [comment]
added multistep forecasting, also new update to hyperparameter at single step to optimize the training dan validation loss.
